/**
 * 生成 grpc 文件
 * protoc -I="." --grpc_out="." --plugin=protoc-gen-grpc="/bin/grpc_cpp_plugin" "message.proto"
 * 生成 pb 文件
 * protoc --cpp_out="." "message.proto"
 **/
syntax = "proto3";

package message;

service VerifyService {
  rpc GetVerifyCode (GetVerifyReq) returns (GetVerifyRsp) {}
}

message GetVerifyReq {
  string email = 1;
}

message GetVerifyRsp {
  int32 error = 1;
  string email = 2;
  string code = 3;
}
